.github/actions/verify-badge/action.yaml

```diff
@@ -0,0 +1,257 @@
+name: Verify-Badge
+description: Perform verification to assist with UDS badge certification
+
+runs:
+  using: composite
+  steps:
+    - name: Test installing the package
+      shell: bash
+      run: |
+        set -uo pipefail
+
+        # Validate UDS Package/Config chart is valid
+
+        echo "ℹ️ Installing kubectl-validate..."
+        go install sigs.k8s.io/kubectl-validate@latest
+
+        echo "ℹ️ Validating uds-config chart..."
+        kubectl validate <(helm template chart chart/)
```

Comment:
can we make chart an input (some packages are monorepos and while `chart` should be the default it isn't where it is everywhere.